resources/views/register.blade.php

```diff
@@ -23,12 +23,12 @@
                     required
                 />
 
-                <x-form.input
+                <livewire:username-input
                     name="username"
                     :label="__('Username')"
                     :value="old('username')"
                     :placeholder="__('Enter your username')"
-                    required
+                    required=""
```

Comment:
Minor detail: wouldn't `:required="true"` work? It'd be more clear I think

Reply:
Fixed